### PR TITLE
Sylwia/moves images

### PR DESCRIPTION
**Pull request by sylwiavargas**
# PR Description

This PR fixes a broken link of an asset (button).

### Summary of my changes and explanation of my decisions

- I moved all the button images to the `/public/img` so they can be accessed by our users and referenced in the docs.

### Self-check

Please check all that apply:

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my code or content update and edited it to the best of my abilities
- [x] I have commented my code, particularly in hard-to-understand areas; my comments are concise

⚡️ ⚡️ ⚡️  Thank you for contributing to StackBlitz ⚡️ ⚡️ ⚡️

### CODE REVIEW

**Comment by fvsch** [comment]
Should we remove the "Edit in Web Publisher" buttons that use the Codeflow icon, and only keep the ones that have the Web Publisher icon?

File: docs/codeflow/integrating-web-publisher.md [hunk 1]
```diff
@@ -111,8 +111,8 @@ To help your users easily find their way to Web Publisher on your site or repo,
 
 | Button preview | Direct URL |
 | --- | --- |
-| <img alt="Edit in Web Publisher" src="./assets/button-edit-in-web-publisher-icon-medium.svg" /> | <a href="./assets/button-edit-in-web-publisher-icon-medium.svg" target="_blank">button-edit-in-web-publisher-icon-medium.svg</a> |
-| <img alt="Edit in Web Publisher" src="./assets/button-edit-in-web-publisher-icon-small.svg" /> | <a href="./assets/button-edit-in-web-publisher-icon-small.svg" target="_blank">button-edit-in-web-publisher-icon-small.svg</a> |
+| <img alt="Edit in Web Publisher" src="/img/button-edit-in-web-publisher-icon-medium.svg" /> | <a href="/img/button-edit-in-web-publisher-icon-medium.svg" target="_blank">button-edit-in-web-publisher-icon-medium.svg</a> |
+| <img alt="Edit in Web Publisher" src="/img/button-edit-in-web-publisher-icon-small.svg" /> | <a href="/img/button-edit-in-web-publisher-icon-small.svg" target="_blank">button-edit-in-web-publisher-icon-small.svg</a> |
```
Comment (fvsch) [inline docs/codeflow/integrating-web-publisher.md:115]: So applying the same naming convention would give us:

```
edit_in_web_publisher.svg
edit_in_web_publisher_small.svg
```

(I'd avoid the `_medium` suffix for the default size. See the `open_in_stackblitz.svg` image for precedent.)

File: docs/codeflow/using-pr-new.md [hunk 2]
```diff
@@ -50,8 +50,8 @@ To help your users quickly spin up the entire environment with your project, you
 
 | Button preview | Direct URL |
 | --- | --- |
-| <img alt="Open in Codeflow" src="./assets/button-open-in-codeflow-medium.svg" /> | <a href="./assets/button-open-in-codeflow-medium.svg" target="_blank">button-open-in-codeflow-medium.svg</a> |
-| <img alt="Open in Codeflow" src="./assets/button-open-in-codeflow-small.svg" /> | <a href="./assets/button-open-in-codeflow-small.svg" target="_blank">button-open-in-codeflow-small.svg</a> |
+| <img alt="Open in Codeflow" src="/img/button-open-in-codeflow-medium.svg" /> | <a href="/img/button-open-in-codeflow-medium.svg" target="_blank">button-open-in-codeflow-medium.svg</a> |
+| <img alt="Open in Codeflow" src="/img/button-open-in-codeflow-small.svg" /> | <a href="/img/button-open-in-codeflow-small.svg" target="_blank">button-open-in-codeflow-small.svg</a> |
```
Comment (fvsch) [inline docs/codeflow/using-pr-new.md:54]: Proposed file names:

```
open_in_codeflow.svg
open_in_codeflow_small.svg
```

Comment (sylwiavargas) [reply docs/codeflow/using-pr-new.md:54]: I see, this makes sense.

File: docs/codeflow/integrating-web-publisher.md (hunk 1 above)
Comment (sylwiavargas) [reply docs/codeflow/integrating-web-publisher.md:115]: This makes sense. Thank you!

**Review by fvsch — APPROVED** [review]
Love it!

**Pull request merged**